repos/fdbt-site/cypress_tests/cypress/e2e/capped.cy.ts

```diff
@@ -0,0 +1,26 @@
+import { isFinished } from '../support/helpers';
+import {
+    defineUserTypeAndTimeRestrictions,
+    selectFareType,
+    completeCappedGeoZonePages,
+    completeCappedDistanceJourney,
+    completeCappedMultiServicePages,
+} from '../support/steps';
+
+
+describe('the capped faretype product journey', () => {
+    it('completes successfully for geozone distance journey', () => {
+        selectFareType('cappedProduct', false);
+        defineUserTypeAndTimeRestrictions();
+        completeCappedGeoZonePages()
+        completeCappedDistanceJourney()
+    });
+
+    it('completes successfully for multi-service distance journey', () => {
+        selectFareType('cappedProduct', false);
+        defineUserTypeAndTimeRestrictions();
+        completeCappedMultiServicePages();
+        completeCappedDistanceJourney()
+        isFinished();
```

Comment:
isFinished() should either be called at the end of every test, or at the end of no tests.